en/scripting/modules/example-protobufjs.md

```diff
@@ -0,0 +1,120 @@
+# Example: Using protobuf.js with npm
+
+This section illustrates how to use the npm package in a Cocos Creator project by installing and using [protobufjs](https://www.npmjs.com/package/protobufjs).
+
+## Install Node.js
+
+The npm package management tool `npm` is included in the Node.js distribution. You can use it after installing Node.js.
+
+Confirm that the installation of `npm` is successful with the following command:
+
+```bash
+> npm -v
+# Possible output:
+# 6.14.9
+```
+
+## Install the npm Package
+
+Executing the following command in the project root directory:
+
+```bash
+> npm install --save protobufjs
+```
+
+Will install the npm package `protobufjs` into the `/node_modules` directory and write the dependencies for this package to the file `package.json`.
+
+The `package.json` file is an npm manifest file and needs to be included in version control.
```

Comment:
an npm manifest file -> the npm manifest file?

Reply:
"an" is correct